app/assets/stylesheets/course_progress.css

```diff
@@ -72,3 +72,8 @@ details[open] .icon-up-arrow-green {
 
 }
 
+.btn-disabled {
```

Comment:
This is not specific to course. It should not be added to this file. Background color should also be changed

Reply:
moved to new css file